desc/objectives/_geometry.py

```diff
@@ -527,10 +536,11 @@ class PlasmaVesselDistance(_Objective):
         Bounding surface to penalize distance to.
     target : {float, ndarray}, optional
         Target value(s) of the objective. Only used if bounds is None.
-        Must be broadcastable to Objective.dim_f.
+        Must be broadcastable to Objective.dim_f. Defaults to ``bounds=(1,np.inf)``.
```

Comment:
Same as my previous questions. Can we make defaults smarter?

Reply:
This one is fairly sensible I think, it's just saying we want the vessel to be at least 1m away from the plasma. There will always be a tradeoff between how complicated the default is vs how useful it is. In 99% of cases, the user should be setting a target/bounds themselves anyways.